BraveRewardsUI/Publisher/PublisherSummaryView.swift

```diff
@@ -6,6 +6,11 @@ import UIKit
 
 class PublisherSummaryView: UIView {
   
+  lazy var monthlyTipView = DetailActionableRow().then {
+    $0.textLabel.text = Strings.TipSiteMonthly
+    $0.batValueView.amountLabel.text = "5"
```

Comment:
Sorry, missed this last review. This shouldn't be set at all here.

Reply:
Moved to WalletViewController.swift